extract/lambda_function.py

```diff
@@ -0,0 +1,204 @@
+import json
+import copy
+import boto3
+import requests
+import pandas as pd 
+from datetime import datetime 
+from io import StringIO
+
+class WeatherData:
+    def __init__(self):
+        self.base_url = "http://api.weatherapi.com/v1/current.json"
+        self.api_key = "98350e503e094a6aae7190947231211"
```

Comment:
Hide this key.

Reply:
done

Reply:
right now i'm offline....i'll revert you back monday